metadata-db-app/src/cmr/metadata_db/data/oracle/concepts.clj

```diff
@@ -644,6 +644,31 @@
                 []
                 concept-id-rev-ids-map))))
 
+(defn get-collection-concept-ids
+  [db provider granule-concept-ids]
+  (cond
+    (empty? granule-concept-ids)
+    []
+
+    :else
+    (let [table (tables/get-table-name provider :granule)
+          ;; Oracle caps IN clause at 1000, so chunk the IDs
+          ;; This is only theoretical, it is unlikely we will get close to even 100.
```

Comment:
Just FYI I've seen plenty of EDSC queries that will send 2000 granule concept-ids, so it's good that you're handling it here.